geth/jail/jail_test.go

```diff
@@ -115,6 +116,44 @@ func (s *JailTestSuite) TestFunctionCall() {
 	require.Equal(expectedResponse, response)
 }
 
+func (s *JailTestSuite) TestJailRPCAsyncSend() {
+	require := s.Require()
+	var wg sync.WaitGroup
+	txSendAsync := `
+		_status_catalog.commands.send({
+			"from": "` + TestConfig.Account1.Address + `",
+			"to": "` + TestConfig.Account2.Address + `",
+			"value": "0.000001"
+		})
+	`
+	rxSendJS := LoadFromFile("../../static/testdata/jail/tx-send/rx-send.js")
```

Comment:
Nope, this should be taken from bindata as an asset. See how this is handled in other tests requiring similar external files.
You won't even need an extra assertion below when you fix this.

Why is it called `rxSendJS`, by the way?